frontend/test/pytest/test_device_api.py

```diff
@@ -103,7 +103,7 @@ def test_qjit_device():
     with EvaluationContext(EvaluationMode.QUANTUM_COMPILATION) as ctx:
         transform_program, _ = device_qjit.preprocess(ctx)
     assert transform_program
-    assert len(transform_program) == 1
+    assert len(transform_program) == 2
```

Comment:
I would suggest to check that the verification really happens last
```suggestion
    assert len(transform_program) == 2
    assert transform_program[-1]._transform.__name__ == "verify_operations"
```